src/bio.c

```diff
@@ -396,18 +395,18 @@ void bioDrainWorker(int job_type) {
  * to perform a fast memory check without other threads messing with memory. */
 void bioKillThreads(void) {
     int err;
-    unsigned long j;
 
-    for (j = 0; j < BIO_WORKER_NUM; j++) {
-        if (bio_threads[j] == pthread_self()) continue;
-        if (bio_threads[j] && pthread_cancel(bio_threads[j]) == 0) {
-            if ((err = pthread_join(bio_threads[j],NULL)) != 0) {
+    for (size_t j = 0; j != BIO_WORKER_NUM; ++j) {
```

Comment:
Perhaps we can keep it of the old type so that we don't need to touch the two lines of logs below.

Reply:
I just want to avoid some changes that doesn't relate this PR itself. BIO_WORKER_NUM is rather small, so there is actually no difference between using unsigne long and size_t.